packages/wp-comments/src/index.ts

```diff
@@ -0,0 +1,153 @@
+import { fetch, warn, URL } from "frontity";
+import WpComments from "../types";
+
+const wpComments: WpComments = {
+  name: "@frontity/wp-comments",
+  state: {
+    comments: {
+      forms: {},
+    },
+  },
+  actions: {
+    comments: {
+      submit: ({ state, actions }) => async (postId, comment) => {
+        // Check first if the connected source is a WP.com site.
+        if (state.source.isWpCom) {
+          return warn(
+            "Sending comments to a WordPress.com site is not supported yet."
+          );
+        }
+
+        // Return if a submission is pending.
+        if (state.comments.forms[postId]?.submitted?.isPending) {
+          return warn(
```

Comment:
Should we return false so that the caller knows it didn't work? Or are we just relying on state update/re-rendering?

Reply:
We are relying on state update/re-rendering here because actions don't return anything. We try to follow the flux pattern so what actions only do is to modify the state. This is how actions work in Frontity by design.

Well, for asynchronous actions, they actually return a Promise that resolves once the action has finished, but it's just an empty Promise.